src/admin/menu/views/MenuEdit.tsx

```diff
@@ -23,6 +23,32 @@ import {
 	MenuEditParams,
 } from '../menu.types';
 
+// TODO: Remove once typings 2.10.0 is released.
+export type ContentPickerTypeSchema =
+	| 'CONTENT_PAGE'
+	| 'COLLECTION'
+	| 'ITEM'
+	| 'BUNDLE'
+	| 'DROPDOWN'
+	| 'INTERNAL_LINK'
+	| 'EXTERNAL_LINK';
+
+export interface MenuSchema {
+	id: number;
+	label: string;
+	icon_name: string;
+	description: string | null;
+	// tslint:disable-next-line: prefer-array-literal
+	user_group_ids: Array<number | string> | { [key: string]: string } | null;
```

Comment:
Dit klopt niet, we zullen enkel number[] hierin opslaan
Dan mag je ook die ignore tslint line wegdoen

![image](https://user-images.githubusercontent.com/1710840/73644013-d9a76b80-4674-11ea-994f-c3aebefc4551.png)